### PR TITLE
[entropy_src] Keep applying fw_ov_rd_fifo_overflow

### PR DESCRIPTION
**Pull request by moidx**
Keep applying fw_ov_rd_fifo_overflow instead of pulsing. This commit is a manual port of the following Opentitan commit:

b454878f2528b49318ff2b604cb8d59a6e59b0ea

Which is associated with
https://github.com/lowRISC/opentitan/pull/21640.

Original commit description:

This commit applies the fw_ov_rd_fifo_overflow signal in case of an overflow. It used to only send a pulse through the register but now the signal is applied until the FIFO has been emptied once and is receiving contiguous data again.
This commit also aligns the documentation for the FW_OV_RD_FIFO_OVERFLOW register. The register is now not clearable by software. It instead is cleared by hardware.

Fixes: https://github.com/chipsalliance/caliptra-rtl/issues/503

### CODE REVIEW

**Comment by howardtr** [comment]
@moidx mentioned that he's going to change main -> dev-goog once we sync up #505

**Review by calebofearth — DISMISSED** [review]

**Comment by calebofearth** [comment]
@howardtr the Microsoft pipeline passed on my clone of this branch. Can you do the following to get the workflow to pass?
- in local copy: bash $CALIPTRA_ROOT/.github/scripts/stamp_repo.sh
- commit the changes to pr_timestamp and pr_hash
- push here.
- I can reapprove.

**Comment by howardtr** [comment]
> @howardtr the Microsoft pipeline passed on my clone of this branch. Can you do the following to get the workflow to pass?
> 
> * in local copy: bash $CALIPTRA_ROOT/.github/scripts/stamp_repo.sh
> * commit the changes to pr_timestamp and pr_hash
> * push here.
> * I can reapprove.

@moidx - can you try running the stamp_repo.sh as recommended by caleb?  We should be able to push to main after, thanks!

**Review by howardtr — DISMISSED** [review]

**Comment by moidx** [comment]
Hi @howardtr, @calebofearth. I have updated the hash following the instructions you provided. Can you approve running the github actions? Thanks!

File: src/entropy_src/data/entropy_src.rdl [hunk 1]
```diff
@@ -688,8 +688,8 @@ addrmap entropy_src {
             in the Observe FIFO. The RNG data rate is slow enough that firmware should always
             be able to keep up. This register meanwhile provides an additional check to confirm
             that bytes read from the !!FW_OV_RD_DATA register represent contiguous RNG samples.
-            If an overflow event occurs, this bit must be cleared by software.";
-            sw = rw;
+            If an overflow event occurs, this bit is cleared by hardware as soon as the FIFO is emptied.";
+            sw = r;
             onwrite = woclr;
```
Comment (calebofearth) [inline src/entropy_src/data/entropy_src.rdl:693]: We'll need to remove the onwrite property

Comment (moidx) [reply src/entropy_src/data/entropy_src.rdl]: Fixed.

**Comment by moidx** [comment]
Hi @calebofearth, I just resolved the previous comment. Thanks

**Comment by calebofearth** [comment]
> Hi @calebofearth, I just resolved the previous comment. Thanks

There was an issue with the pre-run-check that caused it to fail on the compile.yml check job for this PR (because it's from a forked repository). I just merged a PR to the main branch with a fix for the the pre-run-check.yml workflow. 
Can you rebase this branch onto origin/main, then rerun stamp_repo.sh?

**Comment by moidx** [comment]
> Can you rebase this branch onto origin/main, then rerun stamp_repo.sh?

Done

**Review by calebofearth — APPROVED** [review]

**Review by howardtr — APPROVED** [review]

**Pull request merged**